app/helpers/articles_helper.rb

```diff
@@ -13,7 +13,7 @@ def social_link_for article, site
     when :twitter
       "https://twitter.com/intent/tweet?text=#{url_encode article.title}&amp;url=#{article_url}&amp;via=crimethinc"
     when :facebook
-      "https://www.facebook.com/dialog/feed?app_id=966242223397117&display=popup&caption=#{url_encode article.title}&link=#{article_url}&redirect_uri=#{article_url}"
+      "https://www.facebook.com/sharer?u=#{url_encode article_url}"
```

Comment:
is it possible to do something like `url_encode article_short_url || article_url`

Reply:
Yeah, while walking back home I was wondering if it could be possible to do something like that.

Reply:
@just1602 ahh, I see. then yea, this makes more sense